libs/core/src/components/pds-select/stories/pds-select.docs.stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Story, Canvas, Source, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+import * as stories from './pds-select.stories.js';
+
+<Meta
+  title="components/Select"
+  component="pds-select"
+  argTypes={extractArgTypes('pds-select')}
+  args={{
+    checked: false,
+    disabled: false,
+    invalid: false,
+    required: false,
+  }}
+  subcomponents={{
```

Comment:
Just so you know: Last week Storybook was updated and subcomponents in this way are no longer supported. Happy to provide additional info if needed, but we've not agreed on an alternative yet.